uwblueprint/community-fridge-kw
# feat: send 24 hour email reminder for schedule

## Brief description. What is this change? 

### [Ticket Name](https://www.notion.so/uwblueprintexecs/Team-Hub-Phase-1-Enhancements-Phase-2-cf7744a5b04748059761c1649a61a463)




## Implementation description. How did you make this change?

<img width="707" alt="Screen Shot 2022-02-15 at 7 23 39 PM" src="https://user-images.githubusercontent.com/41586370/154172620-e82a5cb3-e77f-417f-8b8b-aa60fae59903.png">



## Steps to test
1. Change `cron.schedule("0 0 0 * *", async () => {` in `cronService.ts` to `cron.schedule("*/10 * * * * *", async () => {`
2. Create a schedule for the next day
3. Check email

P.S.: The email will be sent every 10 seconds so change the cron conditions back to original to avoid spam.


## Checklist
